packages/playground/wordpress/src/index.ts

```diff
@@ -131,27 +147,42 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 			if (!$user) {
 				return;
 			}
+			/**
+			 * This approach is described in a comment on
+			 * https://developer.wordpress.org/reference/functions/wp_set_current_user/
+			 */
 			wp_set_current_user( $user->ID, $user->user_login );
 			wp_set_auth_cookie( $user->ID );
 			do_action( 'wp_login', $user->user_login, $user );
 			setcookie('playground_auto_login_already_happened', '1');
-		}
 
+			/**
+			 * Reload page to ensure the user is logged in correctly.
+			 * WordPress uses cookies to determine if the user is logged in,
+			 * so we need to reload the page to ensure the cookies are set.
+			 *
+			 * Both WordPress home url and REQUEST_URI may include the site scope
+			 * subdirectory.
+			 * To prevent the redirect from including two scopes, we need to
+			 * remove the scope from the REQUEST_URI if it's present.
```

Comment:
How can we remove this special case? WordPress sites installed in a subdirectory seem to support redirects just fine, e.g. `https://w.org/playground`. What are they doing differently than this function?

Reply:
The `home_url` will always include the scope subdirectory.
`REQUEST_URI` might sometimes include it and other times not. I need to check why it's not consistent.

Reply:
It's possible to make unscoped requests to PHP using the request handler.

For example[ this unit test](https://github.com/WordPress/wordpress-playground/blob/97f58114a15663dbd7841150ba8b0e546b0ffbd0/packages/playground/blueprints/src/lib/steps/login.spec.ts#L84) will make an unscoped request to PHP.

We could update the test to include a scope, but it's also something a user might do, so I think it's better to cover that case.

Reply:
I documented it in this PR https://github.com/WordPress/wordpress-playground/pull/1971/files#diff-8afd615c3847003caebf2b35102f3a2a107ad9647e095495ecc9302862802191R166